corehq/apps/userreports/reports/builder/forms.py

```diff
@@ -326,48 +344,87 @@ def __init__(self, app_id, source_type, report_source_id, *args, **kwargs):
         self.helper.attrs['data_bind'] = "submit: submitHandler"
         self.helper.form_id = "report-config-form"
 
-        self.helper.layout = crispy.Layout(
-            self.top_fieldset,
-            FormActions(
-                crispy.ButtonHolder(
-                    crispy.Submit(
-                        'submit',
-                        _(self.button_text)
+        buttons = [crispy.Submit('submit', _(self.button_text))]
```

Comment:
I believe `_(variable)` doesn't actually work, because of how django marks values for translation in the .mo & .po files. I think of the '_' as part of the "translatable string literal" syntax, kinda like the `u` in `u'my string'` is part of the "unicode string literal" syntax (even though this isn't literally true in how python treats the two). Replacing `'Done'` above with `_('Done')` would be a preferable way to do this. If you didn't because that fails, use ugettext_lazy instead.